api-operator/pkg/controller/api/api_controller.go

```diff
@@ -237,6 +237,11 @@ func (r *ReconcileAPI) Reconcile(request reconcile.Request) (reconcile.Result, e
 	}
 
 	owner := getOwnerDetails(instance)
+	if errOwnerSet := setApiDependent(&r.client, instance, &owner); errOwnerSet != nil {
+		log.Error(errOwnerSet, "Error setting owner ref for API dependent configs")
```

Comment:
Can we use complete work for `ref`? So it will help to identify the error message.